### PR TITLE
Update MS SQL Installation instructions

### PR DESCRIPTION
**Pull request by SammyIsConfused**
I was installing openEQUELLA via MS SQL for the first time and got confused with enabling connections via JDBC URL. When getting to the installation step that required database info it kept refusing to connect, even when the database details were absolutely correct. 
The step that we got stuck on was enabling TCP/IP when it is disabled by default in Windows, so in order to capture that I've added explicit instructions on how to accomplish this. 
I also tested installing via Ubuntu, and that was a very quick and painless installation and worked well with openEQUELLA - so I've provided links to the Microsoft documentation on how to do this, via Ubuntu and via Docker.

### CODE REVIEW

File: guides/InstallationAdminGuide.md [hunk 1]
```diff
@@ -47,16 +47,33 @@ openEQUELLA uses the default Users Tablespace that must have an <unlimited> Quot
...
 ```sql
 ALTER DATABASE MyEquellaDatabase SET READ_COMMITTED_SNAPSHOT ON;
 ```
 
+It is now possible to install Microsoft SQL Server for Linux, or as a Docker image. 
+
+[Follow these steps to get it running on Ubuntu.](https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-ubuntu?view=sql-server-ver15)
+
+[Follow these steps to get it running in a Docker Image.](https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-docker?view=sql-server-ver15&pivots=cs1-cmd)
+
```
Comment (edalex-ian) [inline guides/InstallationAdminGuide.md]: This seems at a bit of loose end here (and maybe it's also questionable whether it should be in this guide, but I think it's okay). So to try and bring it together maybe I suggest this alternative:

```suggestion
#### Microsoft SQL Server on Linux

Historically installing Microsoft SQL Server was limited to Windows. However it is now possibly to
run it on linux either directly or with official docker images. Microsoft provide the [SQL Server on
Linux documentation](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-overview) which
includes quick start guides for:

* Installing on
  [RedHat](https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-red-hat?view=sql-server-ver15),
  [SUSE](https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-suse?view=sql-server-ver15)
  or
  [Ubuntu](https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-ubuntu?view=sql-server-ver15);
  and
* Running as a [docker
  image](https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-docker?view=sql-server-ver15&pivots=cs1-bash).
```

**Review by edalex-ian — COMMENTED** [review]
Excellent, really useful @SammyIsConfused .

One suggestion though for the new doco about linux., Whatcha think?

**Comment by SammyIsConfused** [comment]
@edalex-ian Fixed that up now, thanks!

**Review by edalex-ian — APPROVED** [review]
Thanks @SammyIsConfused :+1: :bow:

**Pull request merged**